frontend/src/pages/TermPlanner/LetterGradeButton/LetterGradeButton.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Button } from "antd";
+
+const LetterGradeButton = ({ letterGrade, setInputBuffer, handleConfirm }) => {
```

Comment:
Since `LetterGradeButton` is only used once, could just include it inside the `EditMarks` folder rather than making a seperate component folder for it :)

Reply:
Can definitely move the folder. If it's only being used once, should we just remove the component all together and just move the lettergrade responsibility entirely inside `EditMarks`?

Reply:
Yep that would be better i reckon :)